client/extensions/wp-super-cache/accepted-filenames.jsx

```diff
@@ -18,216 +18,172 @@ import FormToggle from 'components/forms/form-toggle/compact';
-							</span>
-						</FormToggle>
-
-						<FormToggle
-							checked={ !! frontpage }
-							disabled={ isRequesting || isSaving }
-							onChange={ handleAutosavingToggle( 'frontpage' ) }>
-							<span>
-								{ translate( 'Front Page (is_front_page)' ) }
-							</span>
-						</FormToggle>
-
-						<FormToggle
-							checked={ !! home }
-							disabled={ isRequesting || isSaving }
-							onChange={ handleAutosavingToggle( 'home' ) }>
-							<span>
-								{ translate( 'Home (is_home)' ) }
-							</span>
-						</FormToggle>
-
-						<FormToggle
-							checked={ !! archives }
-							disabled={ isRequesting || isSaving }
-							onChange={ handleAutosavingToggle( 'archives' ) }>
-							<span>
-								{ translate( 'Archives (is_archive)' ) }
-							</span>
-						</FormToggle>
-
-						<FormToggle
-							checked={ !! tag }
-							disabled={ isRequesting || isSaving }
-							onChange={ handleAutosavingToggle( 'tag' ) }>
-							<span>
-								{ translate( 'Tags (is_tag)' ) }
-							</span>
-						</FormToggle>
-
-						<FormToggle
-							checked={ !! category }
-							disabled={ isRequesting || isSaving }
-							onChange={ handleAutosavingToggle( 'category' ) }>
-							<span>
-								{ translate( 'Category (is_category)' ) }
-							</span>
-						</FormToggle>
-
-						<FormToggle
-							checked={ !! feed }
-							disabled={ isRequesting || isSaving }
-							onChange={ handleAutosavingToggle( 'feed' ) }>
-							<span>
-								{ translate( 'Feeds (is_feed)' ) }
-							</span>
-						</FormToggle>
-
-						<FormToggle
-							checked={ !! search }
-							disabled={ isRequesting || isSaving }
-							onChange={ handleAutosavingToggle( 'search' ) }>
-							<span>
-								{ translate( 'Search Pages (is_search)' ) }
-							</span>
-						</FormToggle>
-
-						<FormToggle
-							checked={ !! author }
-							disabled={ isRequesting || isSaving }
-							onChange={ handleAutosavingToggle( 'author' ) }>
-							<span>
-								{ translate( 'Author Pages (is_author)' ) }
-							</span>
-						</FormToggle>
-					</FormFieldset>
-
-					<FormFieldset>
+class AcceptedFilenames extends Component {
+	static propTypes = {
+		fields: PropTypes.object,
+		handleChange: PropTypes.func.isRequired,
+		handleSubmitForm: PropTypes.func.isRequired,
+		isRequesting: PropTypes.bool,
+		isSaving: PropTypes.bool,
+		setFieldValue: PropTypes.func.isRequired,
+		translate: PropTypes.func.isRequired,
+	};
+
+	static defaultProps = {
+		fields: {},
+		isRequesting: true,
+		isSaving: false,
+	};
+
+	renderToggle = ( fieldName, fieldLabel ) => {
+		const {
+			fields: { pages },
+			isRequesting,
+			isSaving,
+			translate,
+		} = this.props;
+
+		return (
+			<FormToggle
+				checked={ !! pages && !! pages[ fieldName ] }
+				disabled={ isRequesting || isSaving }
+				onChange={ this.handleToggle( fieldName ) }>
+				<span>
+					{ translate( '%(label)s)', { args: { label: fieldLabel } } ) }
```

Comment:
Good eye! Fixed in https://github.com/Automattic/wp-calypso/pull/13646/commits/73e18dc95827bfd05349810e242bf58ecfaa74f7.